log/src/index.ts

```
@@ -38,6 +39,25 @@ const repo = getRepo(context);
 
+const SUPPORTED_PROTOCOLS = {
+    HTTP: 'http://',
+    HTTPS: 'https://',
+    TCP: 'tcp://'
+} as const;
+
+function isValidUrl(url: string): boolean {
+    try {
+        if (url.startsWith(SUPPORTED_PROTOCOLS.TCP)) {
+            const [host, port] = url.replace(SUPPORTED_PROTOCOLS.TCP, '').split(':');
+            return Boolean(host && port && !isNaN(parseInt(port, 10)));
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)